lib/dotloop/section.rb

```diff
@@ -0,0 +1,60 @@
+module Dotloop
+  class Section
+    FIXED_SECTIONS = %w(
+      contract_dates contract_info financials geographic_description listing_information
+      offer_dates property_address property referral
+    ).freeze
+
+    def self.call(data)
+      new.call(data)
+    end
+
+    def call(data)
+      @sections = { contacts: [] }
+      fix_hash_keys(data).each { |item| build_section(item[0], item[1]) }
+      @sections
+    end
```

Comment:
It's strange to use the 'Service' style call here.  It seems forced- why not just have an initializer?